Bio/codonalign/__init__.py

```diff
@@ -34,32 +34,31 @@ def build(pro_align, nucl_seqs, corr_dict=None, gap_char='-', unknown='X',
 
     Arguments:
      - pro_align  - a protein MultipleSeqAlignment object
-     - nucl_align - an object returned by SeqIO.parse or SeqIO.index
+     - nucl_seqs - an object returned by SeqIO.parse or SeqIO.index
        or a collection of SeqRecord.
      - alphabet   - alphabet for the returned codon alignment
      - corr_dict  - a dict that maps protein id to nucleotide id
      - complete_protein - whether the sequence begins with a start
        codon
-     - frameshift - whether to apply frameshift detection
 
     Return a CodonAlignment object
-
-    >>> from Bio.Alphabet import IUPAC
+    
+    This example answers this Biostarts question: https://www.biostars.org/p/89741/
+    >>> from Bio.Alphabet import IUPAC, generic_protein
```

Comment:
Can you put a blank line here please? Partly for consistent style, but also IIRC, one of the API docstring processing tools wants this too.